st2reactor/st2reactor/garbage_collector/config.py

```diff
@@ -46,38 +48,48 @@ def _register_common_opts():
 
 def _register_garbage_collector_opts():
     logging_opts = [
-        cfg.StrOpt('logging', default='conf/logging.garbagecollector.conf',
-                   help='Location of the logging configuration file.')
+        cfg.StrOpt(
+            'logging', default='conf/logging.garbagecollector.conf',
+            help='Location of the logging configuration file.')
     ]
+
     CONF.register_opts(logging_opts, group='garbagecollector')
 
     common_opts = [
-        cfg.IntOpt('collection_interval', default=DEFAULT_COLLECTION_INTERVAL,
-                   help='How often to check database for old data and perform garbage collection.'),
-        cfg.FloatOpt('sleep_delay', default=DEFAULT_SLEEP_DELAY,
-                     help=('How long to wait / sleep (in seconds) between collection of different '
-                           'object types.'))
+        cfg.IntOpt(
+            'collection_interval', default=DEFAULT_COLLECTION_INTERVAL,
+            help='How often to check database for old data and perform garbage collection.'),
+        cfg.FloatOpt(
+            'sleep_delay', default=DEFAULT_SLEEP_DELAY,
+            help='How long to wait / sleep (in seconds) between '
+                 'collection of different object types.')
     ]
+
     CONF.register_opts(common_opts, group='garbagecollector')
 
     ttl_opts = [
-        cfg.IntOpt('action_executions_ttl', default=None,
-                   help=('Action executions and related objects (live actions, action output '
-                         'objects) older than this value (days) will be automatically deleted.')),
-        cfg.IntOpt('action_executions_output_ttl', default=7,
-                   help=('Action execution output objects (ones generated by action output '
-                         'streaming) older than this value (days) will be automatically deleted.')),
-        cfg.IntOpt('trigger_instances_ttl', default=None,
-                   help=('Trigger instances older than this value (days) will be automatically '
-                         'deleted.'))
+        cfg.IntOpt(
+            'action_executions_ttl', default=None,
+            help='Action executions and related objects (live actions, action output '
+                 'objects) older than this value (days) will be automatically deleted.'),
```

Comment:
Maybe make it explicit what the default value does. I assume it never automatically deletes them?

Same with `action_executions_output_ttl` and `trigger_instances_ttl` below.